pkg/controller/podgroup.go

```diff
@@ -231,6 +236,10 @@ func (s *SchedulerPluginsCtrl) newPodGroup(mpiJob *kubeflow.MPIJob) metav1.Objec
 		scheduleTimeoutSec = schedPolicy.ScheduleTimeoutSeconds
 	}
 	minMember := calculateMinAvailable(mpiJob)
+	var minResources corev1.ResourceList
+	if origin := s.calculatePGMinResources(minMember, mpiJob); origin != nil {
```

Comment:
beyond the scope of this PR: why is `calculatePGMinResources` returning a pointer to ResourceList? ResourceList is already a map, which is already a pointer.

Reply:
Sounds good, let's leave it.